go/consensus/tendermint/tendermint.go

```diff
@@ -898,16 +899,12 @@ func (t *tendermintService) lazyInit() error {
 	tenderConfig.TxIndex.Indexer = "null"
 	tenderConfig.P2P.ListenAddress = viper.GetString(CfgCoreListenAddress)
 	tenderConfig.P2P.ExternalAddress = viper.GetString(cfgCoreExternalAddress)
-	// Convert persistent peer IDs to lowercase (like other IDs) since
-	// Tendermint stores them in a map and uses a case sensitive string
-	// comparison to check ID equality.
-	tenderConfig.P2P.PrivatePeerIDs = strings.ToLower(strings.Join(viper.GetStringSlice(CfgP2PPrivatePeerID), ","))
+	tenderConfig.P2P.PexReactor = !viper.GetBool(CfgP2PDisablePeerExchange)
```

Comment:
Should configuring a node to use a sentry also disable the PEX reactor automatically or is there a reason why you would want to have it enabled?

Reply:
i think that we currently don't have (or need) an explicit `tendermint.sentry.enabled` flag. So it would just be a duplicated flag of the `CfgP2PDisablePeerExchange`.

We could do it based on the `worker.registration.sentry.address` flag, but that assumes that the registration worker is enabled (which currently always is). But i image there being a case where someone would handle registration differently, and wouldn't even need to set the sentry flags.

But yeah if the `worker.registration.sentry.address` is indeed set, then we could automatically disable pex, but just feels a bit to magic and maybe explicitly disabling it might be better.

Reply:
Oh I see, ok then it is fine as-is.